src/Glob.php

```diff
@@ -458,7 +457,7 @@ public static function toRegEx($glob, $flags = 0, $delimiter = '~')
      */
     public static function getStaticPrefix($glob, $flags = 0)
     {
-        if (!Path::isAbsolute($glob) && false === strpos($glob, '://')) {
+        if (!Path::isAbsolute((string) $glob) && false === strpos($glob, '://')) {
```

Comment:
Not a big fan of these string casts, but they are here for BC - previously, an exception would be thrown, while now a `TypeError` raises if the cast is not here.

We can drop them if we move to native type hints.

Reply:
Rolling as-is for now, while #28 will expose most of this stuff later 👍